third_party/terraform/website/docs/r/service_networking_connection.html.markdown

```diff
@@ -49,3 +49,11 @@ The following arguments are supported:
 * `reserved_peering_ranges` - (Required) Named IP address range(s) of PEERING type reserved for
   this service provider. Note that invoking this method with a different range when connection
   is already established will not reallocate already provisioned service producer subnetworks.
+
+## Import 
+Service networking connection can be imported in following ways
+
+* terraform import google_service_networking_connection.peering_connection {{peering-network}}:{{servicenetworking.googleapis.com}}
```

Comment:
```suggestion
* terraform import google_service_networking_connection.peering_connection {{network}}:{{service}}
```

`peering-network` refers to the `network` field value and `servicenetworking.googleapis.com` to the `service` value, right? In that case, we'd format this like so. `{{field}}` indicates that it follows the same format as the field in config, generally.